dspace-api/src/main/resources/spring/spring-dspace-addon-import-services.xml

```diff
@@ -23,8 +23,11 @@
           lazy-init="false" autowire="byType" destroy-method="destroy">
         <property name="importSources">
             <list>
-                <ref bean="PubmedImportService"></ref>
-                <ref bean="BibtexImportService"></ref>
+                <ref bean="PubmedImportService" />
```

Comment:
On this bean & all of the below, spring has guidelines where the bean identifiers should start with a lower case letter.